graphql_compiler/tests/snapshot_tests/test_cost_estimation.py

```diff
@@ -1324,87 +1323,87 @@ class IntegerIntervalTests(unittest.TestCase):
 
     def test_interval_creation(self) -> None:
         """Test that intervals are created correctly, and that empty intervals are detected."""
-        interval = _create_integer_interval(5, 1000)
-        self.assertTrue(interval is not None)
+        interval = Interval[int](5, 1000)
```

Comment:
This is fine and no need to change it now, though I suspect the `[int]` wasn't actually necessary since the integers in the constructor already eliminate all other possibilities for the type parameter.